lib/teleterm/daemon/config.go

```diff
@@ -69,6 +70,14 @@ type Config struct {
 	ConnectMyComputerNodeJoinWait     *connectmycomputer.NodeJoinWait
 	ConnectMyComputerNodeDelete       *connectmycomputer.NodeDelete
 	ConnectMyComputerNodeName         *connectmycomputer.NodeName
+
+	ClientCache ClientCache
+}
+
+type ClientCache interface {
```

Comment:
What's the purpose behind this interface?

Reply:
It is useful for testing, so we can provide a fake cache that doesn't call `ConnectToProxy`. Actually, it is not used in this PR, but in the subsequent one.